experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -28,27 +28,15 @@ import {
 import { LazySequence } from "../lazyField";
 import { FieldKey } from "../../../core";
 import { LazyObjectNode, getBoxedField } from "../lazyTree";
-import { ProxyField, ProxyNode, SharedTreeList, SharedTreeObject } from "./types";
+import {
+	ProxyField,
+	ProxyNode,
+	SharedTreeList,
+	SharedTreeObject,
+	getTreeNode,
+	setTreeNode,
+} from "./types";
 import { getFactoryContent } from "./objectFactory";
-import { createNodeApi, nodeSym } from "./node";
-
-/** Symbol used to store a private/internal reference to the underlying editable tree node. */
-const treeNodeSym = Symbol("TreeNode");
```

Comment:
These were moved to types.ts to prevent a module cycle between `node.ts` and `proxies.ts` (since node.ts now needs to call `getProxyForNode()`)